README.md

```diff
@@ -1,20 +1,76 @@
+
+### Start back-end server locally
+
+Start Phoenix endpoint with `mix phx.server`
+
+## Front-End
+
+### Installation 
+
+Install Node.js dependencies with : `cd assets && npm install`
+
+### Launch front server locally
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-Ready to run in production? Please [check our deployment guides](http://www.phoenixframework.org/docs/deployment).
+## Makefile targets
+
+Commands below will be used for both stacks (backend & workers):
+
+| Command                     | Description                                                                                |
+|-----------------------------|--------------------------------------------------------------------------------------------|
+| `make docker-build`         | Build locally a docker image                                                               |
+| `make docker-clean`         | Remove locally the built docker image                                                      |
+| `make docker-push-registry` | Push the locally built docker image                                                        |
+| `make up`                   | Launch both, the back and front end locally [http://localhost:4000](http://localhost:4000) |
 
-## Learn more
+## Environment variables
+
+Some variables are defined to apply a custom setting. These variables are:
+
+| Variable name          | Default value                | Description                                     |
+|------------------------|------------------------------|-------------------------------------------------|
+| `AMQP_MANAGEMENT_PORT` |                              | Port used to query RabbitMQ and get admin data. |
+| `DATABASE_NAME`        |                              | Name of the database used.                      |
+| `DATABASE_PORT`        |                              | Port of the database used.                      |
+| `DATABASE_PASSWORD`    |                              | Database password.                              |
+| `DATABASE_USERNAME`    |                              | Database login.                                 |
+| `DATABASE_HOSTNAME`    |                              | Host of the database (IP or DNS).               |
+| `PORT`                 |                              | Port to access the application.                 |
+| `SSL`                  |                              | If TRUE the application run with HTTPS.         |
+| `ROOT_PASSWORD`        | `admin123`                   | Application password for administrator.         |
+| `ROOT_EMAIL`           |                              | Application email for administrator.            |
+| `HOSTNAME`             |                              | Application hostname (or IP).                   |
+
+More environment variables are available due to using library [rs_amqp_worker](https://github.com/media-cloud-ai/rs_amqp_worker).
```

Comment:
we don't use rs_amqp_worker here.

Reply:
A bad copy/paste because variables are the same for AMQP. But these name are not linked. So, I copy used variable in this file.
Done.